File: FrontEnd-ProjectReact/my-react-app-inventory/src/types/location.ts
import type { Warehouse } from "./warehouse";
import type { ItemDetail } from "./itemDetail";

export interface Location {
  id: number;
  locationCode: string;
  wareHouseCode: string;
  name: string;
  modifiedDateTime: Date;
  createdDateTime: Date;
  status?: string;

  wareHouseCodeNavigation: Warehouse;
  itemDetail?: ItemDetail;
}

File: BackEnd-ProjectAPI/Sample/Sample.Infrastructure/EntityFramework/Location.cs
using System;
using System.Collections.Generic;

namespace Sample.Infrastructure.EntityFramework
{
    public partial class Location
    {
        public int Id { get; set; }
        public string LocationCode { get; set; } = null!;
        public string? Name { get; set; }
        public string WarehouseCode { get; set; } = null!;
        public string? Remarks { get; set; }
        public DateTime ModifiedDateTime { get; set; }
        public DateTime CreatedDateTime { get; set; }
        public string? RecStatus { get; set; }

        public virtual Warehouse WarehouseCodeNavigation { get; set; } = null!;
    }
}
